api/cosmos/streaming/abci/v1/grpc.pulsar.go

```diff
@@ -0,0 +1,4248 @@
+// Code generated by protoc-gen-go-pulsar. DO NOT EDIT.
```

Comment:
This file (`api/cosmos/streaming/abci/v1/grpc.pulsar.go`) and `api/cosmos/streaming/abci/v1/grpc_grpc.pb.go` still need to be deleted and `make proto-gen` needs to be rerun too.

```shell
$ rm -rf api
$ make proto-gen
$ go mod tidy
```

Reply:
removed, regenerated proto and tidied up.